cmake/Modules/CUDAQConfig.cmake

```diff
@@ -44,11 +44,39 @@ if (NOT CUDAQ_LIBRARY_MODE)
   enable_language(CUDAQ)
 endif() 
 
+# ---- TARGET EXPORTS ----
+add_library(cudaq::cudaq-nvidia-target SHARED IMPORTED)
+set_target_properties(cudaq::cudaq-nvidia-target PROPERTIES
+  IMPORTED_LOCATION "${CUDAQ_LIBRARY_DIR}/libnvqir-custatevec-fp32${CMAKE_SHARED_LIBRARY_SUFFIX}"
```

Comment:
Do we need updates for the libnvqir-cusvim-* targets, too?

Reply:
Addressed in latest commit. Let me know what you think